modMcf/test/org/aion/mcf/account/FileDateTimeComparatorTest.java

```diff
@@ -133,4 +134,29 @@ public void testComparatorWithGarbage() {
             }
         }
     }
+
+    @Test
```

Comment:
I would like to see more edge case tests. Timestamps, especially a non-trivial one like this, can be prone to small errors. See what happens when you give a month with 3 digits, or two times that differ by the absolute smallest quantity the timestamp allows for, etc.